### ISSUE TITLE
Splash screen has an area for version but it is blank.

### ISSUE THREAD

**Issue by rrousselot** [dbeaver/dbeaver#16166]
#### System information: 
- Operating system (distribution) and version  Microsoft Windows [Version 10.0.19042.1415]
- DBeaver version  Version 22.0.2.202204041900
- Additional extensions

#### Connection specification:
N/A

#### Describe the problem you're observing:
Splash screen has an area for version but it is blank.

#### Steps to reproduce, if exist:
Start software.

#### Include any warning/errors/backtraces from the logs
N/A

**Comment by LonwoLonwo** [comment]
Hello @rrousselot 

Can you provide a video of the issue, please?
Usually, the version info does not occur immediately, but after a while after the appearance of the splash screen.

**Comment by XiozTzu** [comment]
https://share.icloud.com/photos/080WL_Nu2m92rCrg-wVsSBfcQ

Hope that link works. This is a PC Win 10. Tried on Mac M1 and it worked but just barely.

**Comment by LonwoLonwo** [comment]
Sorry, your video is unvailable right now.
Can you please check the newest DBeaver version please?
https://dbeaver.io/download/

**Comment by uslss** [comment]
there has been no update on the issue for a long time.
If it is still actual - feel free to reopen it with additional info.

**Comment by maffe** [comment]
I wanted to check the version number and thought the splash screen would be a good place to look for it. But since it only showed the string “VERSION” without an actual version number, I assumed it was broken and did something else while DBeaver was starting up.
![grafik](https://github.com/user-attachments/assets/85904369-e4bb-435f-9ed0-543f4c67a806)
Later I found out the version number will be displayed in the splash screen, but only after several seconds.
![grafik](https://github.com/user-attachments/assets/c7007e88-bf1f-4a74-861e-e2888ca2ccfa)

Some ways to avoid displaying the string “VERSION” without the actual version number:
- Don’t show splash screen until version number is available
- Generate image with version number at build time
- Omit the string “VERSION” from the image and move it to the text overlay (that way it would also use the same font and color)

I also noticed the dynamic text at the bottom (displaying “Loading” in the second screenshot) is partially covered by the progress bar.

Environment: Windows 10, 4K display with 200 % zoom, DBeaver 24.2.3.
